src/WebJobs.Script.WebHost/Routing/ScriptRouteHandler.cs

```diff
@@ -71,6 +71,10 @@ public ScriptRouteHandler(ILoggerFactory loggerFactory, ScriptHost scriptHost)
             bool authorized = await AuthenticateAndAuthorizeAsync(context, descriptor);
             if (!authorized)
             {
+                if (descriptor.Metadata.IsDisabled)
```

Comment:
This was logic that currently lives [here](https://github.com/Azure/azure-webjobs-sdk-script/blob/v1.x/src/WebJobs.Script.WebHost/FunctionRequestInvoker.cs#L113) in v1.x. Are you saying this wasn't carried forward in any way? That concerns me - what else was missed. We had end to end tests (sample tests) that would have caught this, but in the core branch all those tests are deleted/disabled. @fabiocav ?

Reply:
isDisabled for httpTrigger was missed in the .net core port.

Reply:
This logic still doesn't seem correct, and doesn't appear to be replicating the v1.x behavior. That behavior is: if a function is disabled it can be invoked if an only if we're admin authorized. It looks like here, if the request IS authorized, you won't be doing the disabled check?

I think getting the correct IsDisabled behavior back in place should be done in a separate commit, and to the point where the existing tests (currently commented out/deleted?) are passing again. I had a bunch of tests for this. Maybe @fabiocav can help with this, since he did the port and might have more input on it.

Reply:
As discussed I reverted this code and created an item assigned to @fabiocav:
Add new middleware to handle "dissabled" state for httpTrigger #212